<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propMaterial</class>
 <widget class="QWidget" name="propMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>122</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>122</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>122</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="material">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>2</y>
     <width>194</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Material</string>
   </property>
   <widget class="QGroupBox" name="colorGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Front Color</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="editRGBA">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>58</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>#rrggbbaa</string>
     </property>
     <property name="text">
      <string>#00000000</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSlider" name="sliderG">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>28</y>
       <width>96</width>
       <height>9</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>green component</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderA">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>48</y>
       <width>96</width>
       <height>9</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>alpha component</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderB">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>38</y>
       <width>96</width>
       <height>9</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>blue component</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sliderR">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>18</y>
       <width>96</width>
       <height>9</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>red component</string>
     </property>
     <property name="accessibleName">
      <string>R</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="preGrey">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>37</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(164, 164, 164);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="preRed">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>18</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="preBlack">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>18</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(96, 96, 96);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="preCyan">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>18</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="preBlue">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>56</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="preMagenta">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>37</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 255 );</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="preYellow">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>56</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="preWhite">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>56</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(224, 224, 224);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="preGreen">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>37</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkShow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>show instead of v-/f- colors</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propMaterial</receiver>
   <slot>rValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propMaterial</receiver>
   <slot>gValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propMaterial</receiver>
   <slot>bValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderA</sender>
   <signal>valueChanged(int)</signal>
   <receiver>propMaterial</receiver>
   <slot>aValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editRGBA</sender>
   <signal>textChanged(QString)</signal>
   <receiver>propMaterial</receiver>
   <slot>rgbaTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preBlack</sender>
   <signal>clicked()</signal>
   <receiver>propMaterial</receiver>
   <slot>colorButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preBlue</sender>
   <signal>clicked()</signal>
   <receiver>propMaterial</receiver>
   <slot>colorButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preCyan</sender>
   <signal>clicked()</signal>
   <receiver>propMaterial</receiver>
   <slot>colorButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preGreen</sender>
   <signal>clicked()</signal>
   <receiver>propMaterial</receiver>
   <slot>colorButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preGrey</sender>
   <signal>clicked()</signal>
   <receiver>propMaterial</receiver>
   <slot>colorButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preMagenta</sender>
   <signal>clicked()</signal>
   <receiver>propMaterial</receiver>
   <slot>colorButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preRed</sender>
   <signal>clicked()</signal>
   <receiver>propMaterial</receiver>
   <slot>colorButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preWhite</sender>
   <signal>clicked()</signal>
   <receiver>propMaterial</receiver>
   <slot>colorButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preYellow</sender>
   <signal>clicked()</signal>
   <receiver>propMaterial</receiver>
   <slot>colorButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkShow</sender>
   <signal>stateChanged(int)</signal>
   <receiver>propMaterial</receiver>
   <slot>checkShowClicked(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rValueChanged(int)</slot>
  <slot>gValueChanged(int)</slot>
  <slot>bValueChanged(int)</slot>
  <slot>aValueChanged(int)</slot>
  <slot>rgbaTextChanged(QString)</slot>
  <slot>meshColorChanged(QColor)</slot>
  <slot>colorButtonClicked()</slot>
  <slot>checkShowClicked(int)</slot>
 </slots>
</ui>
